pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -246,6 +247,12 @@ let
             broken = brokenOpen;
           })
           openSha256;
+        efa-nv-peermem = lib.mapNullable
+          (nvidia-open: callPackage ../efa/nv-peermem.nix {
+            inherit nvidia-open;
+            broken = brokenEFANVPeerMem;
+          })
+          self.open;
```

Comment:
It's a nit and it's certainly OK for the first iteration, but I think making the open driver a passthru attribute of `nvidia_x11` might have been a mistake in the first place, and maybe we're repeating it.

Reply:
Yeah, as I wrote on matrix, I really don't like that this has to be here... but as of today this codesmell is (IMO) a lesser evil than the footgun of having efa-nv-peermem exist in top-level.nix with an abstract nvidia_x11 argument (which would default to nvidiaPackages.stable unless explicitly overridden).

I can take a look at pulling it out, but I don't know what to expect in terms of breakage and rebuilds... I want this module present when boot-testing changes I want to make on libfabric/nccl, and if untangling nvidia-open is messy, I'd rather handle it as a follow-up.

On the other hand, adding a new attribute here already causes some mass rebuilds, so maybe it's a horse apiece.

Reply:
> I can take a look at pulling it out

I'd suggest we move it out later (if at all), together with the open driver